far/FarEng.hlf.m4

```diff
@@ -6874,6 +6874,22 @@ deleted, or renamed.
  This parameter can be changed via ~far:config~@FarConfig@ only.
 
 
+@Panel.TreatDotFilesAsHidden
+$ #far:config Panel.TreatDotFilesAsHidden#
+ This Boolean parameter controls the filtering of files or folders
+starting with a dot (“dotfiles”).
+ ^<wrap>It works in combination with the #Show hidden and system files and folders#
```

Comment:
It is better to use exact phrase as it appears in the dialog (and in the help above): `#Show hidden and system files#`. The "files and folder" wording is more precise, and we should probably adopt it, but we should be consistent.

Reply:
Thanks!